deploy/evaluation/Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG GO_VER=1.21
+ARG PY_VER=3.11
+
+FROM golang:${GO_VER} AS builder
+ARG GOPROXY=https://goproxy.cn,direct
+
+# build arctl
+WORKDIR /go/src
+COPY . .
```

Comment:
Please only copy the required packages:

COPY cmd cmd
COPY pkg/ pkg/
COPY go.mod go.mod
COPY go.sum go.sum